src/application_config/app_config.h

```diff
@@ -18,29 +47,67 @@
  * @addtogroup SDK15
  */
 /** @{ */
-/**
- * @file app_config.h
- * @author Otso Jousimaa <otso@ojousima.net>
- * @date 2020-04-16
- * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
- *
- */
+
+#ifndef APP_FW_NAME
+#   define APP_FW_NAME "Ruuvi FW"
+#endif
+
+#ifndef APP_FW_VERSION
+#define APP_FW_VERSION  "v3.0.0"  // Overridden by 'make' provided define: -DAPP_FW_VERSION ; from git describe --tags
```

Comment:
This is defined elsewhere as v0.0.1, please remove

Reply:
I thought it was a good idea to 
1) identify this as version 3
2) show where it comes from.

Anyway it is preceded by  <pre>#ifdef</pre>